js/coinflex.js

```diff
@@ -0,0 +1,1885 @@
+            'status': status,
+            'side': side,
+            'price': finalLimitPrice,
+            'type': orderType,
+            'stopPrice': this.safeNumber (order, 'stopPrice'),
+            'amount': this.safeNumber (order, 'quantity'),
+            'filled': this.parseNumber (filledQuantityRaw),
+            'remaining': this.safeNumber2 (order, 'remainQuantity', 'remainingQuantity'),
+            'average': this.parseNumber (avgPriceRaw),
+            'cost': this.parseNumber (cost),
+            'fee': fees,
+            'trades': trades,
+            'info': order,
+        }, market);
+    }
+
+    async fetchPosition (symbol, params = {}) {
+        await this.loadMarkets ();
+        const positions = await this.fetchPositions (undefined, params);
+        const array = this.filterBySymbol (positions, symbol);
+        return this.safeValue (array, 0); // exchange doesn't seem to have hedge mode, so the array will contain only one position per symbol
+    }
+
+    async fetchPositions (symbols = undefined, params = {}) {
+        const data = await this.getAccountData (params);
+        // response sample inside `getAccountData` method
+        this.targetAccount = this.safeValue (data, 0);
+        const positions = this.safeValue (this.targetAccount, 'positions', []);
+        return this.parsePositions (positions);
+    }
+
+    parsePositions (positions) {
+        const result = [];
+        for (let i = 0; i < positions.length; i++) {
+            result.push (this.parsePosition (positions[i]));
+        }
+        return result;
+    }
+
+    parsePosition (position, market = undefined) {
+        //
+        //     {
+        //         "marketCode": "SHIB-USD-SWAP-LIN",
+        //         "baseAsset": "SHIB",
+        //         "counterAsset": "USD",
+        //         "position": "313546.0",
+        //         "entryPrice": "0.00002161",
+        //         "markPrice": "0.00002158",
+        //         "positionPnl": "-0.009406380",
+        //         "estLiquidationPrice": "0",
+        //         "lastUpdatedAt": "1651342025876"
+        //     }
+        //
+        //    but this.targetAccount has also like:
+        //
+        //     {
+        //         "collateral": "28.297558",
+        //         "notionalPositionSize": "6.82903188",
+        //         "portfolioVarMargin": "0.682589",
+        //         "riskRatio": "41.5216",
+        //         "maintenanceMargin": "0.34129450",
+        //         "marginRatio": "1.20",
+        //         "liquidating": false,
+        //         "feeTier": "0",
+        //         "createdAt": "1651232948406"
+        //     }
+        //
+        const marketId = this.safeString (position, 'marketCode');
+        market = this.safeMarket (marketId, market);
+        const contractsString = this.safeString (position, 'position');
+        const timestamp = undefined; // this.safeInteger (position, 'lastUpdatedAt');
```

Comment:
Is anything wrong with this timestamp?

Reply:
this is not order's timestamp, but as name suggests also "last updated" timestamp, which can have different values (last triggered trade of order, last modified event of order, or whatever), so doesn't relate to timestamp of order.